java-play-framework/2.5.x/README.md

```diff
@@ -0,0 +1,65 @@
+[<img src="https://img.shields.io/travis/playframework/play-java-starter-example.svg"/>](https://travis-ci.org/playframework/play-java-starter-example)
+
+# rookout and play-java-starter-example
+
+This is a starter application that shows how Play works with Rookout.
+
+## Run Rookout Agent 
+
+1. Run the Rookout agent:
+    ``` bash
+    $ docker run -p 7486:7486 -e "ROOKOUT_TOKEN=<Your-Token>" rookout/agent
+    ```
+## Running with sbt dist
+
+1.  Run the following commands:
+    ``` bash
+    $ cd 2.5.x
+    $ sbt dist
+    $ cd target/universal
+    $ unzip play-java-starter-example-1.0-SNAPSHOT.zip 
+    $ cd play-java-starter-example-1.0-SNAPSHOT/bin
+    ```
+
+    - For Windows run:
+    ``` bash
+    $ play-java-starter-example.bat -Dplay.file=../conf/application.conf
+    ```
+
+    - For Unix run:
+    ``` bash
+    $ chmod +x play-java-starter-example
+    $ ./play-java-starter-example -Dplay.file=../conf/application.conf
+    ```
+
+## Rookout Integration explained
+This example make sure to set up Rookout`s SDK as Java Agent for Play Framework
+  1. Add Java Agent plugin
+    ```
+    addSbtPlugin("com.lightbend.sbt" % "sbt-javaagent" % "0.1.4")
+    ```
+  1. Enable Java Agent
+    ```
+    lazy val root = (project in file(".")).enablePlugins(PlayJava, JavaAgent)
+    ```
+  1. Set rook as the Java Agent
+    ```
+    javaAgents += "com.rookout" % "rook" % "0.1.9" % "dist"
+    ```
+
+## Running with sbt run
+Create lib folder and download Rookout`s rook module
```

Comment:
Not sure i understand this

Reply:
You can run with sbt run or package it with sbt dist... the 2 options are familiar for those who use sbt